### PR TITLE
Improved comments and bugfixes (SafeConfigParser, division by zero, log.warn)

### PR DESCRIPTION
**Pull request by Boldfor**
1. Improved comments to BS440.py.
2. Bug around `SafeConfigParser` is fixed. With `SafeConfigParser` (in current version) the following error is thrown:

```
/home/pi/BS440-HA-AD/BS440_commented.py:286: DeprecationWarning: The SafeConfigParser class has been renamed to ConfigParser in Python 3.2. This alias will be removed in future versions. Use ConfigParser directly instead.
  config = SafeConfigParser()
```
ConfigParser is used instead.
3. Division by zero (in case user is not identified) fixed
4. Error message around log.warn fixed
```
/home/pi/BS440-HA-AD/BS440_commented.py:376: DeprecationWarning: The 'warn' method is deprecated, use 'warning' instead
  log.warn('Error getting handles')
```
5. Improved section "Run the script at startup"-section in README.md

### CODE REVIEW

File: BS440.py [hunk 1]
```diff
@@ -53,35 +84,34 @@ def decodePerson(handle, values):
...
+        10    -pad byte-                                      x (pad byte) 
+        11    -pad byte-                                      x (pad byte) 
+        12    -pad byte-                                      x (pad byte) 
+        13    person                       [1..8]             B (integer, length 1)
+        --> Interpretation pattern:                           BHxxIxxxxB              
+    '''
     data = unpack('<BHxxIxxxxB', bytes(values[0:14]))
     retDict = {}
     retDict["valid"] = (data[0] == 0x1d)
+    # Weight is reported in g. Hence, divide by 100.0
```
Comment (creatvty) [inline BS440.py]: I'm not sure anymore where the information is from, but converting weight from `g` to `kg` would require division by `1000` :smile: . So Conversely I'd say the precision is actually 10g

File: BS440.py [hunk 2]
```diff
@@ -91,44 +121,98 @@ def decodeWeight(handle, values):
...
+        Some scales (e.g. BS 444) identify the max. 8 users via a "weight approximation".
+        If a new (or guest) user stands on the scale, the scale is not able to identify the user.
+        As a result, no size for the user is stored in the scale, and thus not transmitted
+        to the script. In these situations, the script stopped (division by zero). The following
+        check aims at solving this problem (https://github.com/keptenkurk/BS440/issues/102)
+        '''
+        if size != 0:
+            element['bmi'] = round(element['weight'] / (size * size), 1)
+        else:
+            element['bmi'] = 0
```
Comment (creatvty) [inline BS440.py]: You can simplify this a little
```suggestion
        if size == 0:
            element['bmi'] = 0
        else:
            element['bmi'] = round(element['weight'] / (size * size), 1)
```

**Review by creatvty — COMMENTED** [review]
Hey, thanks for the improvements. I can't testify too much on the `README.md` changes since it has been a while but it looks sense making to me.
About the code changes, I left some comments but it looks good to me. I tried it out and will continue to try it out. But for now it seems to work.

File: BS440.py (hunk 1 above)
Comment (Boldfor) [reply BS440.py]: Good point. Did update the comment. Slipped my mind when I digged through the code.

**Comment by creatvty** [comment]
Just gotta fix the conflicts in Readme now so I can merge

**Review by creatvty — APPROVED** [review]

**Pull request merged**